doc/architecture/decisions/0008-data-manipulation-for-views-should-happen-in-util-files.md

```diff
@@ -0,0 +1,23 @@
+# 7. Data manipulation for views should happen in util files
+
+Date: 2023-02-14
+
+## Status
+
+Proposed
+
+## Context
+
+Nunjucks is limited as to how much manipulation it allows and therefore some kinds of neccessary manipulation can only happen in Node.
+There isn't a clear precedent in HMPPS as to where such manipulation should happen.
+We initially did such manipulation in the services but as the app has evolved we now believe that
+
+## Decision
```

Comment:
This is really helpful, thanks.

I think that possibly for people like me who aren't very close to the code it would be important to have an example use-case of the problem? 

Can you include some code snippets e.g. "before" and "after" to illustrate the 2 approaches and make the differences more concrete?